theme.json

```diff
@@ -253,41 +253,50 @@
 				}
 			],
 			"fontSizes": [
+				{
+					"fluid": {
+						"max": "0.813rem",
+						"min": "0.688rem"
```

Comment:
I would keep this one non fluid since it's small as it is. Just let it be 0.8 on any screen size